google-cloud-dlp-v2/Rakefile

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-# Copyright 2020 Google LLC
+# Copyright 2021 Google LLC
```

Comment:
The copyright year changed because 2021 is the copyright year in https://github.com/googleapis/googleapis-gen/tree/master/google/privacy/dlp/v2/google-cloud-privacy-dlp-v2-ruby.

Preserving the copyright year would require adding a post-processor.  Would you like to add a post processor?

Reply:
Yes, we absolutely need to add a copyright year preserving post-processor before putting this in production.

Reply:
Understood.  Thanks.